test-utils/src/test_env.rs

```diff
@@ -508,8 +510,8 @@ pub async fn create_state_merkle_tree_and_queue_account<R: RpcConnection>(
             merkle_tree_config.clone(),
             queue_config.clone(),
             program_owner,
+            forester,
             index,
```

Comment:
this seems off? 

function signature says: 

pub fn create_initialize_merkle_tree_instruction(
    payer: Pubkey,
    registered_program_pda: Option<Pubkey>,
    merkle_tree_pubkey: Pubkey,
    nullifier_queue_pubkey: Pubkey,
    state_merkle_tree_config: StateMerkleTreeConfig,
    nullifier_queue_config: NullifierQueueConfig,
    program_owner: Option<Pubkey>,
    index: u64,
    additional_rent: u64,
)